plugins/glustershd/rest.go

```diff
@@ -201,3 +201,90 @@ func selfHealHandler(w http.ResponseWriter, r *http.Request) {
 	}
 	restutils.SendHTTPResponse(ctx, w, http.StatusOK, nil)
 }
+
+func checkBrickExistence(volinfo *volume.Volinfo, hostname string, brickname string) bool {
+	bricks := volinfo.GetBricks()
+	for index := range bricks {
+		if bricks[index].Hostname == hostname && bricks[index].Path == brickname {
+			return true
+		}
+	}
+	return false
+}
+
+func splitBrainOperationHandler(w http.ResponseWriter, r *http.Request) {
+	// Collect inputs from URL
```

Comment:
question: can we have a request body for this API instead of query parameters

Reply:
@Madhu-1 I guess having a request body will make it a lot more easier to use. Will make the changes.